addons/web_editor/views/snippets.xml

```diff
@@ -719,6 +731,19 @@
                 <we-button class="fa fa-fw fa-rotate-right" data-name="img_shape_transform_rotate_y_opt" data-set-img-shape-rotate-right="true"
                     data-no-preview="true" data-dependencies="shape_img_opt" title="Rotate right"/>
             </we-row>
+            <we-row t-if="not no_animations" string="Speed" class="o_we_sublevel_1">
+                <we-range data-dependencies="shape_img_opt" data-name="shape_anim_opt" t-att-string="animation_speed_label"
+                          t-att-class="indent and 'o_we_sublevel_2'"
+                          data-no-preview="true"
+                          data-display-range-value="true"
+                          data-set-animation-ratio=""
+                          data-exponential="true"
+                          data-append="x"
+                          data-max="10"
+                          data-min="-10"
+                          data-step="0.1"
+               />
```

Comment:
Just passing by with codeowner notification by email

Note that there is no guidelines about this, it's always a thought subject.
But I would not add it to the previous line, because adding new attribute means there will be 2 line of diff instead of two

Same rationale as why we do in python:
```py
a = {
    1,
    2,
    3,
}
```
With a trailing `,` after the `3`, so when we add `4`, it will be one line of diff:
```diff
a = {
    1,
    2,
    3,
+   4,
}
```
Not 2:
```diff
a = {
    1,
    2,
-   3
+   3,
+   4,
}
```